test/e2e/storage_performant_policy.go

```diff
@@ -0,0 +1,289 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+
+	g "github.com/onsi/ginkgo/v2"
+)
+
+const (
+	clientName         = "cluster-storage-operator-e2e"
+	storageCOName      = "storage"
+	fsGroupPolicyLabel = "storage.openshift.io/fsgroup-change-policy"
+	selinuxPolicyLabel = "storage.openshift.io/selinux-change-policy"
+)
+
+var (
+	waitPollInterval = time.Second
```

Comment:
Checking audit logs why a namespace is not deleted, 1 second is really noisy there

Reply:
fixed it to 2s and made logs less noisy.